src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");

```

Comment:
The message should indicate what type of 'Machine' is working. Consider specifying the type of machine in a more informative way. For example: 'Truck is starting its work.'